portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -20,13 +20,15 @@
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
+
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    final String myName = "Jess DeJong";
```

Comment:
Having a final in a method (while allowed), is very uncommon. It doesn't change based on any method input, so this should be a class-level static:

`private static final String MY_NAME = "Jess DeJong";`

This goes outside the method right below the class declaration line. Also note the variable name style.

Reply:
Thanks, that makes sense! I'll make the change.